doc/pairs.md

```diff
@@ -0,0 +1,206 @@
+# Pairs examples
+
+With ``crud.pairs``, you can iterate across a distributed space. The arguments are the same as [``crud.select``](https://github.com/tarantool/crud/docs/select.md), except or the ``use_tomap`` parameter. Below are examples that may help you.
+
+## ``Use_tomap`` parameter
+
+With ``use_tomap`` flag, you can choose to iterate over objects or over tuples. If ``use_tomap = false``, you will iterate over tuples. This parameter is false by default.
+
+**Example:**
+
+```lua
+tuples = {}
+for _, tuple in crud.pairs('developers', nil, { use_tomap = false, first = 3 }) do
```

Comment:
I'm not sure we need to show `use_tomap = false` separately.
I would like to show the simplest `pairs` in the first example and then say that you can iterate over objects using `use_tomap` flag.